java/src/main/java/org/eclipse/ditto/client/live/events/internal/ImmutableGlobalEventFactory.java

```diff
@@ -200,6 +200,7 @@ public FeaturePropertyModified featurePropertyModified(final ThingId thingId, fi
     }
 
     @Override
+    @Deprecated
```

Comment:
I think we shouldn't deprecate this method. It is not directly tied to V1 and if we introduce a V3 one day, we need it again.